### PR TITLE
Fix `store` description of picker fields

### PR DESCRIPTION
**Pull request by afbora**
### Fixes
- Picker fields `store` prop description corrected #4780 

### Breaking changes
None

## Ready?

- [x] ~Unit tests for fixed bug/feature~
- [x] In-code documentation (wherever needed)
- [x] Tests and checks all pass

### For review team

- [x] Add changes to release notes draft in Notion
- [x] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)

### CODE REVIEW

File: config/fields/mixins/picker.php [hunk 1]
```diff
@@ -68,13 +69,13 @@
 		},
 
 		/**
-		 * Whether to store UUID or path in
-		 * the content file
+		 * Whether to store UUID in the content file
+		 * or ID of the model
```
Comment (bastianallgeier) [inline config/fields/mixins/picker.php]:
```suggestion
		 * Whether to store UUID or ID in the 
		 * content file of the model
```

**Review by bastianallgeier — DISMISSED** [review]

File: config/fields/mixins/picker.php (hunk 1 above)
Comment (afbora) [reply config/fields/mixins/picker.php]: Does the content file contain the ID data?

**Review by bastianallgeier — APPROVED** [review]

File: config/fields/mixins/picker.php (hunk 1 above)
Comment (bastianallgeier) [reply config/fields/mixins/picker.php]: We could probably just remove the content file part, but what it means is that the UUID or ID is stored in the field in the content file. 

I.e. 

```
related: 

- blog/article-a
- blog/article-b
```

vs. 

```
related: 

- page://blabla
- page://blablabla
```

Comment (afbora) [reply config/fields/mixins/picker.php]: Thanks for reply. I got it what you mean 👍

**Pull request merged**